cypress/integration/pages/testsForAllPages.js

```diff
@@ -44,7 +45,7 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
           .should('have.attr', 'tabindex', '-1');
       });
 
-      if (pageType !== 'errorPage404') {
+      if (!pageTypeSomeTestsNotRun) {
```

Comment:
```suggestion
      if (!pageTypeSomeTestsNotRun(pageType)) {
```

atm this if statement will always return false.

![image](https://user-images.githubusercontent.com/30599794/82034765-85aa1b00-9696-11ea-9913-78f228b8b83a.png)